cdc/redo/writer/file.go

```diff
@@ -495,7 +508,11 @@ func (w *Writer) flushAll() error {
 
 	ctx, cancel := context.WithTimeout(context.Background(), defaultS3Timeout)
 	defer cancel()
-	return w.writeToS3(ctx, w.file.Name())
+
+	err = w.writeToS3(ctx, w.file.Name())
+	w.metricFlushAllDuration.Observe(time.Since(start).Seconds())
```

Comment:
Is this metric S3 only, if using NFS, this metric is not observed.

Reply:
nfs use the same metric as local file 
```
	start := time.Now()
	err = w.file.Sync()
	w.metricFsyncDuration.Observe(time.Since(start).Seconds())
```